packages/graphql-language-service-server/src/GraphQLCache.ts

```diff
@@ -54,8 +54,11 @@ const {
   DIRECTIVE_DEFINITION,
 } = Kind;
 
-export async function getGraphQLCache(configDir: Uri): Promise<GraphQLCache> {
-  const graphQLConfig = await getGraphQLConfig(configDir);
+export async function getGraphQLCache(
+  configDir: Uri,
+  config?: GraphQLConfig,
+): Promise<GraphQLCache> {
+  const graphQLConfig = config || (await getGraphQLConfig(configDir));
```

Comment:
what i was thinking is that if you can pass a config instance here, you might be able to create the graphql-config programatically from an object literal, especially with the new 3.0.0 graphql-config once thats stable

Reply:
Not sure I follow what you mean. Could you expand a bit?

Reply:
my bad, i'm seeing here where 2.0.0 better suits this need, i was mistaken and thought that 1.2.x supported this too, but i see that this is the only option for now